service/src/main/java/uk/nhs/adaptors/oneoneone/cda/report/mapper/ConditionMapper.java

```diff
@@ -13,28 +15,43 @@
 import lombok.AllArgsConstructor;
 import uk.nhs.adaptors.oneoneone.cda.report.util.DateUtil;
 import uk.nhs.adaptors.oneoneone.cda.report.util.NodeUtil;
+import uk.nhs.connect.iucds.cda.ucr.POCDMT000002UK01Component5;
 import uk.nhs.connect.iucds.cda.ucr.POCDMT000002UK01Encounter;
+import uk.nhs.connect.iucds.cda.ucr.POCDMT000002UK01Section;
 
 @Component
 @AllArgsConstructor
 public class ConditionMapper {
 
     private final NodeUtil nodeUtil;
 
-    public Condition mapCondition(POCDMT000002UK01Encounter itkEncounter, Encounter encounter) {
+    public Condition mapCondition(POCDMT000002UK01Section section, POCDMT000002UK01Encounter itkEncounter, Encounter encounter) {
         Condition condition = new Condition();
 
         condition.setIdElement(newRandomUuid());
 
         Date assertedDate = DateUtil.parse(itkEncounter.getEffectiveTime().getValue());
 
+        String languageCode = StringUtils.EMPTY;
+        for (POCDMT000002UK01Component5 component : section.getComponentArray()) {
+            if (component.getSection() != null) {
+                if (component.getSection().isSetLanguageCode()) {
+                    if (component.getSection().getLanguageCode().isSetCode()) {
+                        languageCode = component.getSection().getLanguageCode().getCode();
+                    }
+                }
+            }
+        }
+
         condition
             .setClinicalStatus(Condition.ConditionClinicalStatus.ACTIVE)
             .setVerificationStatus(Condition.ConditionVerificationStatus.UNKNOWN)
             .setSubject(encounter.getSubject())
             .setContext(new Reference(encounter))
-            .setAsserter(encounter.getParticipantFirstRep().getIndividual())
-            .setAssertedDate(assertedDate);
+            .setAssertedDate(assertedDate)
```

Comment:
We should populate full CodeableConcept - system, code and display
See for detailed values: http://hl7.org/fhir/stu3/valueset-languages.html